etc/config-template.toml

```diff
@@ -30,8 +30,12 @@ end-point-concurrency = 8
 # capacity = 0 # 0 is unlimited.
 
 [metric]
-# prometheus client report interval, set 0s to disable prometheus.
-report-interval = "5m"
+# prometheus client push interval, set 0s to disable prometheus.
+interval = "15s"
+# prometheus pushgateway address, leaves it empty will disable prometheus.
```

Comment:
By the way, if I use SIGUSR1 to dump metrics, will it still be pushed to remote? Or the dumped metrics will be excluded to the remote?